src/all_implementations.jl

```diff
@@ -293,10 +293,17 @@ end
 ### EmptyArray ###########################################################
 
 struct EmptyArray{BEHAVIOR} <: LeafType{BEHAVIOR}
-    EmptyArray(; behavior::Symbol = :default) = new{behavior}()
+    behavior::Symbol
+    
+    function EmptyArray(; behavior::Symbol = :default)
+        new{behavior}(behavior)
+    end
 end
```

Comment:
@jpivarski - I changed that to allow a copy beneath. The need for the `behavior::Symbol` showed in a new test.